src/components/Login/index.jsx

```diff
@@ -0,0 +1,165 @@
+import { Alert, Input } from "antd";
+import React, { useState } from "react";
+import { validateEmail } from "../../functions";
+import { useTranslation } from "react-i18next";
+
+const Login = (props) => {
+  const [t] = useTranslation();
+  const [loginCredentials, setLoginCredentials] = useState({
+    email: "",
+    password: "",
+  });
+  const handleChange = (key, value) => {
+    let newValues = Object.assign({}, loginCredentials);
+    newValues[key] = value;
+    setLoginCredentials(newValues);
+  };
+  const [errors, setErrors] = useState({});
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    let { email, password } = loginCredentials;
+    let errors = {};
+    if (!validateEmail(email)) {
+      errors["email"] = "Email format error";
+    }
+    if (email.trim() === "") {
+      errors["email"] = "Please fill this field";
+    }
+    if (password.trim().length < 6) {
+      errors["password"] = "Password must be at least 6 characters long";
+    }
+    if (password.trim() === "") {
+      errors["password"] = "Please fill this field";
+    }
+    setErrors(errors);
+    if (Object.keys(errors).length > 0) {
+      return;
+    }
+    props.onSubmit(loginCredentials);
+  };
+  const handleFacebookAuth = () => {
```

Comment:
in these and the other PRS it's cleaner just to inline these instead of having a one line handle X

Reply:
PTAL